### PR TITLE
NovaCustom battery management

### PR DESCRIPTION
**Pull request by johanes2115**

### CODE REVIEW

File: docs/unified/novacustom/battery-management.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# NovaCustom power and battery subsystem
+
+NovaCustom laptops equipped with Dasharo firmware provide several battery
+management mechanisms to optimize user experience and further prolong the
+battery's lifespan.
+
+## Boot  block when the battery is low
```
Comment (mkopec) [inline docs/unified/novacustom/battery-management.md:7]:
```suggestion
## Block booting when the battery is low
```

File: docs/unified/novacustom/battery-management.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+![](images/battery_connection.jpg)
+
+## Power information error
+
+When the firmware encounters problems with retrieving information about AC and
+battery state, following error message will be displayed:
+
+![](images/power_error.jpg)
+
+## Charging levels
```
Comment (mkopec) [inline docs/unified/novacustom/battery-management.md:31]:
```suggestion
## Charge thresholds
```

File: docs/unified/novacustom/battery-management.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+Dasharo firmware implements various battery charging level limits, which can
+vastly extend the lifespan of the battery:
```
Comment (mkopec) [inline docs/unified/novacustom/battery-management.md:34]:
```suggestion
Dasharo firmware implements battery charge thresholds, which aim to extend the
lifespan of the battery:
```

File: docs/unified/novacustom/battery-management.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+- plugging in the charger when the battery level sits below 95%, will result in
+    charging the battery until 98% of capacity is reached. The battery status
+    will then change to `not charging`
```
Comment (mkopec) [inline docs/unified/novacustom/battery-management.md:38]:
```suggestion
- charging will only start when the battery level is below 95%
```

File: docs/unified/novacustom/battery-management.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+- plugging in the charger when the battery level sits between 95% and 98%, will
+    result in not activating the battery charging mechanism util the battery
+    level below 95% is reached.
```
Comment (mkopec) [inline docs/unified/novacustom/battery-management.md:42]:
```suggestion
- charging will stop once the battery level reaches 98%
    
Custom charge thresholds can be configured using the Dasharo setup menu or using
third-party software like [tlp](https://linrunner.de/tlp/faq/battery.html).
```

Comment (mkopec) [reply docs/unified/novacustom/battery-management.md:42]: The setup menu feature is not implemented or documented yet, but it will be before the release.

**Review by mkopec — CHANGES_REQUESTED** [review]

File: docs/unified/novacustom/battery-management.md (hunk 1 above)
Comment (johanes2115) [reply docs/unified/novacustom/battery-management.md]: Changed

File: docs/unified/novacustom/battery-management.md (hunk 2 above)
Comment (johanes2115) [reply docs/unified/novacustom/battery-management.md]: Changed

File: docs/unified/novacustom/battery-management.md (hunk 3 above)
Comment (johanes2115) [reply docs/unified/novacustom/battery-management.md]: Changed

File: docs/unified/novacustom/battery-management.md (hunk 4 above)
Comment (johanes2115) [reply docs/unified/novacustom/battery-management.md]: Changed

File: docs/unified/novacustom/battery-management.md (hunk 5 above)
Comment (johanes2115) [reply docs/unified/novacustom/battery-management.md]: Changed

**Review by mkopec — APPROVED** [review]

**Pull request merged**